lib/gulp-tasks/file-globbing.js

```diff
@@ -0,0 +1,130 @@
+/**
```

Comment:
I took me few minutes and I had to read all the file to figure out what "globbing" means. Maybe there is a better name? I don't know, like "gulp-inject-task"